docs/user-guide/custom-themes.md

```diff
@@ -118,6 +118,7 @@ favicon           | [site_favicon]        |
 page_description  | [site_description]    |
 repo_url          | [repo_url]            |
 repo_name         | [repo_name]           |
+edit_uri          | [edit_uri]            |
```

Comment:
Let's remove this. We don't actually have a global `edit_uri` variable. You have to access it through the config as `config.edit_uri` and I don't think we need or want to advertise that.